tutorials/networking/high_level_multiplayer.rst

```diff
@@ -235,23 +235,34 @@ You might have already noticed something different, which is the usage of the ``
 
     remote func register_player(info):
 
-This keyword has two main uses. The first is to let Godot know that this function can be called from RPC. If no keywords are added,
-Godot will block any attempts to call functions for security. This makes security work a lot easier (so a client can't call a function
-to delete a file on another client's system).
-
-The second use is to specify how the function will be called via RPC. There are four different keywords:
+This keyword is one of many that allow a function to be called by a remote procedure call (RPC). There are six of them total:
 
 - ``remote``
 - ``remotesync``
-- ``master``
 - ``puppet``
+- ``puppetsync``
+- ``master``
+- ``mastersync``
+
+Each of them designate who can call the rpc, and optionally ``sync`` if the RPC can be called locally.
+
+.. note:: If no rpc keywords are added, Godot will block any attempts to call functions remotely.
+          This makes security work a lot easier (so a client can't call a function to delete a file on another client's system).
+
+The ``remote`` keyword can be called by any peer, including the server and all clients.
+The ``puppet`` keyword means a call can be made from the network master to any network puppet.
+The ``master`` keyword means a call can be made from any network puppet to the network master.
+
+If ``sync`` is included, the call can also be made locally. For example, to allow the network master to change the player's position on all peers:
+
+::
 
-The ``remote`` keyword means that the ``rpc()`` call will go via network and execute remotely.
+    puppetsync func update_position(new_position):
+        position = new_position
 
-The ``remotesync`` keyword means that the ``rpc()`` call will go via network and execute remotely, but will also execute locally (do a normal function call).
+.. tip:: You can also use :ref:`SceneTree.get_rpc_sender_id() <class_SceneTree_method_get_rpc_sender_id>` to have more advanced rules on how an rpc can be called.
 
-The others will be explained further down.
-Note that you could also use the ``get_rpc_sender_id`` function on ``SceneTree`` to check which peer actually made the RPC call to ``register_player``.
+These keywords are further explained in `Synchronizing the game`_.
```

Comment:
Will this relative link work?

Reply:
I think you need to add a reference for it, so, here:
```
:ref:`Synchronizing the game <_doc_high_level_multiplayer_synchronizing>`
```

And where the title is:

```
.. _doc_high_level_multiplayer_synchronizing:

Synchronizing the game
----------------------
```